### ISSUE TITLE
missing_presence_validation: unexpected error on HABTM join tables

### ISSUE THREAD

**Issue by abodelot** [gregnavis/active_record_doctor#67]
Hi!

when running `rake active_record_doctor:missing_presence_validation`, the gem reports a missing validation error on the FK columns on join tables (used in HABTM relations).

Is this really the intended behavior of this gem? Since there is no model for the join table, it shouldn't report an error IMHO.

Example with schema:
```rb
  create_table "pets", force: :cascade do |t|
    t.string "name", null: false
    t.date "birth", null: false
  end

  create_table "pets_users", id: false, force: :cascade do |t|
    t.integer "users_id", null: false
    t.integer "pets_id", null: false
    t.index ["pets_id"], name: "index_pets_users_on_pets_id"
    t.index ["users_id"], name: "index_pets_users_on_users_id"
  end

  create_table "users", force: :cascade do |t|
    t.string "email"
  end
```

Pet:
```rb
  class Pet < ApplicationRecord
    has_and_belongs_to_many :users
    validates :name, presence: true
    validates :birth, presence: true
  end
```

User model
```rb
class User < ApplicationRecord
  has_and_belongs_to_many :pets
end
```

Output:
```
The following models and columns should have presence validations:
  HABTM_Users: users_id, pets_id
  HABTM_Pets: users_id, pets_id
```

Expected output: no error

* gem version: ` active_record_doctor (1.8.0)`

**Comment by gregnavis** [comment]
@abodelot, thanks for the bug report. The fix is on `master` and will be released soon.

**Comment by gregnavis** [comment]
@abodelot, I've just released v1.9.0.rc1 which should address some of your issue. Please add the following to `Gemfile`:

```ruby
gem "active_record_doctor", "= 1.9.0.rc1"
```

and run `bundle install`. Detailed installation and usage instructions are in the README. If you stumble upon any issues then please report them here.

**Comment by gregnavis** [comment]
I'm closing the issue as it's addressed in 1.9.0 which I released a few minutes ago. If there are still issues then feel free to reopen.
